src/plugins/github/README.md

```diff
@@ -0,0 +1,175 @@
+
+A review of a GitHub pull request, e.g. [this review]. Reviews are connected to
+their author(s), to entities they reference, to their comments, and to the pull
+they review. If a positive weight is set on the reviews, then every review will
+mint cred; this could be abused by spammers.
+
+[this review]: https://github.com/sourcecred/sourcecred/pull/91#pullrequestreview-105254836
+
+### Comment
+
+A comment on an issue, pull request, or pull request review. Comments are
+connected to their author(s), to entities they reference, and to their 'parent'
+(the containing issue, pull, or review). If a positive weight is set on
+comments, then every comment will mint cred. This can be absued by spammers,
+and also may encourage long bikeshedding discussions or flame wars.
+
+### Commit
+
+A commit is a Git commit, as discovered via the GitHub API, e.g. [this commit].
+We currently enumerate every commit that is in the history of the `master`
+branch. We added commits back when we had a Git plugin; since we've disabled
+that plugin, commits are no longer well supported. Commits are connected to
+their author(s), to entities they reference, to their parent and child commits,
+and to the pull that merged them (if any). If a positive weight is set on
+commits, then every commit will mint cred. This is not recommended, since
+commits are not well-supported.
+
+Commits will become useful once we re-implement a Git plugin, perhaps with
+directory and file tracking. Until then, we may deprecate them since they don't
+add much to the plugin at the moment.
+
+[this commit]: https://github.com/sourcecred/sourcecred/commit/94b04541514b991c304616aadfcb417a19871e82
+
+### User
+
+A GitHub user account, e.g. [@decentralion]. User accounts do not mint cred, so
+setting a node weight would have no effect. Using the identity plugin, it's
+possible to "collapse" user nodes with other identity nodes into a single,
+canonical identity. For example, if a contributor had a GitHub user account and
+a Discourse account, then the identity plugin can collapse those identities
+together.
+
+Users are connected to posts they author, to posts they react to, and to posts that
+mention them.
+
+[@decentralion]: https://github.com/decentralion
+
+### Bot
+
+A GitHub user account that has been explicitly marked as a bot, via inclusion
+in [bots.js]. This is useful so that we can filter out bot accounts from
+receiving grain or showing up in the cred rankings.
+
+Bots have the same connections as users.
+
+[bots.js]: https://github.com/sourcecred/sourcecred/blob/master/src/plugins/github/bots.js
+
+
+## Edges
+
+### Authors
+
+An authors edge connects an author (i.e. a user or bot) to a post (i.e. a pull,
+issue, comment, or review). If the post contains the text "paired with
+@other-author", then from SourceCred's perspective, that post will have
+multiple authors, all of whom receive an equal share of the cred.
+
+The following are all valid examples of using "paired with":
```

Comment:
Here is an example of co-authored by (sorry if you don't like medium, it can be evil sometimes!) https://medium.com/blackode/creating-co-authored-commits-on-the-command-line-git-305ae2af9f73

Reply:
Yeah, we could definitely update the code to handle co-authored as well as paired-with. Feel free to file an issue (or add the feature directly). :) Grepping for "paired with" in the codebase will help you find where this would be added.

Reply:
Sure thing, I'll open an issue for this (it feels low priority atm).